### PR TITLE
fix: updated common styles of Complete profile

### PR DESCRIPTION
**Pull request by nadiia-tsytsylina**
## Describe your changes
add Back Button to Profile button component;
updated profile page;
add onBack props to 2-6 Complete profile components; 
removed icon from Complete profile header

Снимок экрана 2023-12-06 165308
Снимок экрана 2023-12-06 165351

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/browse/CC-147)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/components/profile/profile-qualification/certificate-form/styles.ts [hunk 1]
```diff
@@ -28,9 +28,14 @@ export const ErrorMessage = styled(Typography)`
   font-weight: ${typography.fontWeightMedium};
 `;
 
+export const ButtonWrapper = styled('div')`
+  border-top: 1px solid var(--divider, rgba(0, 0, 0, 0.12));
```
Comment (Simpsonkk) [inline src/components/profile/profile-qualification/certificate-form/styles.ts:32]: Could you please use a theme or consts for colors?

File: src/components/complete-profile-second/work-form/styles.ts [hunk 2]
```diff
@@ -28,9 +28,14 @@ export const ErrorMessage = styled(Typography)`
   font-weight: ${typography.fontWeightMedium};
 `;
 
+export const ButtonWrapper = styled('div')`
+  border-top: 1px solid var(--divider, rgba(0, 0, 0, 0.12));
```
Comment (Simpsonkk) [inline src/components/complete-profile-second/work-form/styles.ts:32]: Could you please use a theme or consts for colors?

File: src/components/complete-profile-fourth/CompleteProfileFourth.tsx [hunk 3]
```diff
@@ -17,7 +17,13 @@ import {
...
 
-export default function CompleteProfileFourth({ onNext }: { onNext: () => void }): JSX.Element {
+export default function CompleteProfileFourth({
+  onNext,
+  onBack,
+}: {
+  onNext: () => void;
+  onBack: () => void;
+}): JSX.Element {
   const { translate } = useLocales();
```
Comment (Simpsonkk) [inline src/components/complete-profile-fourth/CompleteProfileFourth.tsx:27]: Could you please write type Props in front of the component?

File: src/components/profile/profile-qualification/certificate-form/styles.ts (hunk 1 above)
Comment (nadiia-tsytsylina) [reply src/components/profile/profile-qualification/certificate-form/styles.ts]: fixed

File: src/components/complete-profile-second/work-form/styles.ts (hunk 2 above)
Comment (nadiia-tsytsylina) [reply src/components/complete-profile-second/work-form/styles.ts]: fixed

File: src/components/profile/bio/Bio.tsx [hunk 4]
```diff
@@ -30,8 +29,9 @@ import {
   MOV_FORMAT,
   MP4_FORMAT,
 } from 'src/components/profile/bio/constants';
+import ProfileBtn from 'src/components/reusable/profile-btn/ProfileBtn';
 
-export function Bio(): JSX.Element {
+export function Bio({ onBack }: { onBack: () => void }): JSX.Element {
```
[the remark this answers is not in the document]
Comment (nadiia-tsytsylina) [reply src/components/profile/bio/Bio.tsx]: fixed

**Review by Simpsonkk — APPROVED** [review]

**Review by ProggerOleg — APPROVED** [review]

**Review by lyudad — APPROVED** [review]

**Pull request merged**